### ISSUE TITLE
IFN training data？

### ISSUE THREAD

**Issue by fsigriehgiruewhwruh** [RyuZhihao123/Plant-Recon-25#9]
the ifn trainning script needs training data,  python ./code/train_ifn.py train --data_dir data/ --epochs 300 --batch_size 32, it likes rgb, mask, and tokens, could you provide training data or pretrained checkpoint?

<img width="819" height="160" alt="Image" src="https://github.com/user-attachments/assets/d76c0d2b-126f-43c7-8687-37174969a1bd" />

**Comment by RyuZhihao123** [comment]
Please switch to my another software (https://github.com/RyuZhihao123/Modular-Tree-Modeler-25) to automatically create the plant & structure dataset. This data software was developed by me as well and the free version can be freely download and use on Windows PC.

Demo videos (3 pieces in total): https://www.bilibili.com/video/BV1waLizwEdH

**Comment by fsigriehgiruewhwruh** [comment]
Thank you for your answer, but how do I get tokens?

**Comment by fsigriehgiruewhwruh** [comment]
<img width="1513" height="796" alt="Image" src="https://github.com/user-attachments/assets/26a82326-ce96-4aa3-ba1a-b9a925539fa4" />
How to show the  Box visualization mode

**Comment by RyuZhihao123** [comment]
Q1. Tokens are not generated by the dataset software. The training of IFN is after the training of VAE, andthe tokens are obtained from this step.
Q2. For visualization, please switch to the correct scene file first... You are runing in a so-called "**Untitled Scene**", which is not the main entrance of the program and therefore you can find nothing... Please double click the scene file under [this path](https://github.com/RyuZhihao123/Plant-Recon-25/tree/main/BoxPlantModeling/Assets/Scenes) in Unity. The correct program entrance should be this **SampleScene.unity**.

**Also, the 3D plant figure/screenshot that you provided to me is not the final 3D model.** After you enter the correct entrance scene, press the key "P" and then you will see the truly final 3D plant as shown in the following first figure. Please also refer to the last attached video for details about this (I recorded this video for another person, thus it introduces something unrelated).

<img width="248" alt="Image" src="https://github.com/user-attachments/assets/4b0acc3b-596a-48e2-a610-c0400ea3f178" />

https://github.com/user-attachments/assets/9803e4e9-3d65-4235-9f39-66153f2a66c2

**Comment by fsigriehgiruewhwruh** [comment]
Thanks for your reply, Yes I have entered the correct entrance scene. Are the detailed steps as follows: import the .obj files saved by DataGenerator into Assets, add the Main.cs component to the .obj, and then execute Play? However, how should the several parameters here be configured? I randomly select several, but can't get the result i want

<img width="1920" height="1040" alt="Image" src="https://github.com/user-attachments/assets/e2692528-ec1c-4c90-9c1d-f33469ff8fe3" />

<img width="1918" height="1032" alt="Image" src="https://github.com/user-attachments/assets/15ae7feb-77c5-476d-b0f3-1b94abc5ea08" />

**Comment by RyuZhihao123** [comment]
This issue is most likely caused by **incorrect HDRP rendering settings in the Unity Editor**, rather than a problem with the code implementation. (because this issue is often accompanied by the pink-colored ground plane and white artifacts in the leaf’s alpha-culling areas).

You can search a bit about the tutorial on how to properly use HDRP in unity, which should help you address this issue. 
Usually a re-configuration of HDRP setting can fix this rendering issue easily.

**Comment by fsigriehgiruewhwruh** [comment]
thank you so much! so how to get the Box for trainning the vae
<img width="1920" height="1019" alt="Image" src="https://github.com/user-attachments/assets/073ab8e3-1a9d-4544-8f45-f9f0aa944058" />

<img width="1919" height="1039" alt="Image" src="https://github.com/user-attachments/assets/2ca0bbfa-23f4-46d0-a4eb-30c6e5d70667" />

**Comment by RyuZhihao123** [comment]
I notice you deleted a newly left message when I tried to write new response.
Anyways, you can write me through email (or leave your message through Bilibili's chatbox if you use Bilibili as well), which I believe we can communicate more timely.

**Comment by fsigriehgiruewhwruh** [comment]
Thank you very much! Can we communicate via Bilibili? I've already sent you a message.
